vdr-shared/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "vdr-shared"
+version = "0.1.0"
+authors = ["Hyperledger Indy Contributors <hyperledger-indy@lists.hyperledger.org>"]
+edition = "2018"
+
+description = "This is the official SDK for Hyperledger Indy (https://www.hyperledger.org/projects), which provides a distributed-ledger-based foundation for self-sovereign identity (https://sovrin.org). The major artifact of the SDK is a c-callable library."
```

Comment:
Is indy-vdr intended to be the official SDK for hyperledger indy?

Reply:
For the capabilities it provides - the ledger interface.  So not "the" official SDK, but one of several for different components.

Reply:
might be good to modify this description then